commit d78db12abd9a367644914f8f0c78e688616a4092 (from a447e76fdfecc18a5a93a8be9e99a0560c9cfbe7)
Merge: a447e76 f26f26b
Author: ErikZalm <erik@vdzalm.eu>
Date:   Mon Sep 9 07:09:19 2013 -0700

    Merge pull request #589 from wolfmanjm/upgrade
    
    Fix the example delta configuration by adding DELTA_DIAGONAL_ROD_2

diff --git a/Marlin/example_configurations/delta/Configuration.h b/Marlin/example_configurations/delta/Configuration.h
index cb92365..dbd2892 100644
--- a/Marlin/example_configurations/delta/Configuration.h
+++ b/Marlin/example_configurations/delta/Configuration.h
@@ -82,6 +82,8 @@
 // and processor overload (too many expensive sqrt calls).
 #define DELTA_SEGMENTS_PER_SECOND 200
 
+// NOTE NB all values for DELTA_* values MOUST be floating point, so always have a decimal point in them
+
 // Center-to-center distance of the holes in the diagonal push rods.
 #define DELTA_DIAGONAL_ROD 250.0 // mm
 
@@ -97,6 +99,8 @@
 // Effective horizontal distance bridged by diagonal push rods.
 #define DELTA_RADIUS (DELTA_SMOOTH_ROD_OFFSET-DELTA_EFFECTOR_OFFSET-DELTA_CARRIAGE_OFFSET)
 
+#define DELTA_DIAGONAL_ROD_2 sq(DELTA_DIAGONAL_ROD)
+
 // Effective X/Y positions of the three vertical towers.
 #define SIN_60 0.8660254037844386
 #define COS_60 0.5

